server/data/repositories/star.repository.js

```diff
@@ -8,6 +9,33 @@ class StarRepository extends BaseRepository {
       attributes: ['id', 'userId', 'repositoryId']
     });
   }
+
+  getStars(username) {
+    return this.model.findAll({
+      attributes: [],
+      include: [
+        {
+          model: UserModel,
+          attributes: ['id', 'username'],
+          where: { username }
+        },
+        {
+          model: RepositoryModel,
+          attributes: {
+            include: [
+              [
+                sequelize.literal(`
+                (SELECT COUNT(*)
```

Comment:
Better to use `Sequlize.fn`:
https://www.woolha.com/tutorials/sequelize-aggregate-functions-sum-count-min-max-etc-examples